assets/css/app.scss

```diff
@@ -70,6 +70,7 @@ $vpp-location-padding: 1rem;
 @import "map/markers/missed_stop_icon";
 @import "map/markers/stop_icon";
 @import "map/markers/user_location_marker";
+@import "minimal_ladder_page";
```

Comment:
note: I don't think we'll need this if you do https://github.com/mbta/skate/pull/2588#discussion_r1600363159